files/image_config/monit/container_checker

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+
+"""
+container_checker
+
+This script is intended to be run by Monit. It will write an alerting message into
+syslog if it found containers which were expected to run but were not running. Note
+that if print(...) statement in this script was executed, the string in it will be 
+appended to Monit syslog messages.
+
+The following is an example in Monit configuration file to show how Monit will run 
+this script:
+
+check program container_checker with path "/usr/bin/container_checker"
+    if status != 0 for 5 times within 5 cycles then alert repeat every 1 cycles
+"""
+
+import subprocess
+import sys
+
+import swsssdk
+from sonic_py_common import multi_asic
+
+
+def get_command_result(command):
+    """
+    @summary: This function will execture the command and return the result.
+    @return: A string which contains the output of command.
+    """
+    command_stdout = ""
+
+    try:
+        proc_instance = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE,
+                                         shell=True, universal_newlines=True)
+        command_stdout, command_stderr = proc_instance.communicate()
+        if proc_instance.returncode != 0:
+            print("Failed to execute the command '{}'. Return code: '{}'".format(
+                command, proc_instance.returncode))
+            sys.exit(1)
+    except OSError as os_err:
+        print("Failed to execute the command '{}'. Error: '{}'".format(command, os_err))
+        sys.exit(2)
+    except ValueError as val_err:
+        print("Failed to execute the command '{}'. Error: '{}'".format(command, val_err))
+        sys.exit(3)
```

Comment:
Suggest simplifying this, since the behavior is essentially the same. E.g.,

```python3
except (OSError, ValueError) as err:
        print("Failed to execute the command '{}'. Error: '{}'".format(command, err))
        sys.exit(2)
```